src/opencl_bitcracker_fmt_plug.c

```diff
@@ -0,0 +1,630 @@
+ /*
+ * BitCracker OpenCL version developed by Elenago 
+ * <elena dot ago at gmail dot com> in 2015
+ *
+ * Copyright (c) 2015-2017 Elenago <elena dot ago at gmail dot com>
+ * and Massimo Bernaschi <massimo dot bernaschi at gmail dot com>
+ *
+ * Licensed under GPLv2
+ * This program comes with ABSOLUTELY NO WARRANTY, neither expressed nor
+ * implied. See the following for more information on the GPLv2 license:
+ * http://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * Please contact or cite if you want to use this source code.
+ * More informations at http://openwall.info/wiki/john/OpenCL-BitCracker
+ */
+
+#ifdef HAVE_OPENCL
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_opencl_bitcracker;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_opencl_bitcracker);
+#else
+
+#include <string.h>
+#include "common-opencl.h"
+#include "opencl_device_info.h"
+#include "config.h"
+#include "options.h"
+#include <time.h> 
+
+#define CIPHERTEXT_LENGTH	64
+#define BINARY_SIZE		32
+#define BENCHMARK_COMMENT	""
+#define BENCHMARK_LENGTH	-1
+#define FORMAT_LABEL            "bitcracker-opencl"
+#define FORMAT_NAME             "BitCracker-OpenCL"
+#define ALGORITHM_NAME          "BitCracker OpenCL"
```

Comment:
And shouldn't format label and format name use "BitLocker" instead of "BitCracker"?